csv/src/main/java/com/fasterxml/jackson/dataformat/csv/CsvGenerator.java

```diff
@@ -245,7 +245,7 @@ public CsvGenerator(IOContext ctxt, int jsonFeatures, int csvFeatures,
         _ioContext = ctxt;
         _formatFeatures = csvFeatures;
         _schema = schema;
-        _writer = new CsvEncoder(ctxt, csvFeatures, out, schema);
+        _writer = new CsvEncoder(ctxt, jsonFeatures, csvFeatures, out, schema);
```

Comment:
Let's actually extract the new setting (boolean), pass that, not bit field. I know this may become complicated if more settings were to be passed, but for now this seems slightly cleaner.

Reply:
changed code to pass through a boolean